modules/gitops-release-notes-1-12-2.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-12-2_{context}"]
+= Release notes for {gitops-title} 1.12.2
+
+{gitops-title} 1.12.2 is now available on {OCP} 4.12, 4.13, 4.14, and 4.15.
+
+[id="errata-updates-1-12.2_{context}"]
+== Errata updates
+
+[id="RHSA-XXXX:NNNN-gitops-1-12-2-security-update-advisory_{context}"]
+=== RHSA-XXXX:NNNN - {gitops-title} 1.12.2 security update advisory
+
+Issued: 2024-05-06
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-XXXX:NNNN[RHSA-XXXX:NNNN]
```

Comment:
Same question as above.

Reply:
This is a temporary placeholder for now, @eromanova97